apps/omg_watcher_info/lib/omg_watcher_info/exit_consumer.ex

```diff
@@ -31,16 +31,17 @@ defmodule OMG.WatcherInfo.ExitConsumer do
 
   def init(args) do
     topic = Keyword.fetch!(args, :topic)
+    event_type = Keyword.fetch!(args, :event_type)
     bus_module = Keyword.get(args, :bus_module, @default_bus_module)
 
     :ok = bus_module.subscribe(topic, link: true)
 
     _ = Logger.info("Started #{inspect(__MODULE__)}, listen to #{inspect(topic)}")
-    {:ok, %{}}
+    {:ok, %{event_type: event_type}}
```

Comment:
If the plan is to expand this (eventually) I suggest you hold a struct. If not, keep this value as the state and not a map.

Reply:
I'll keep it simple as I don't foresee expansion of this module. I'll introduce the state var there 👍